test/core/security/security_connector_test.cc

```diff
@@ -340,6 +341,29 @@ static grpc_ssl_roots_override_result override_roots_permanent_failure(
   return GRPC_SSL_ROOTS_OVERRIDE_FAIL_PERMANENTLY;
 }
 
+static void test_ipv6_address_san(void) {
+  const char* addresses[] = {
+      "2001:db8::1",
+      "fe80::abcd:ef65:4321%em0",
+      "fd11:feed:beef:0:cafe::4",
+  };
+  const char* san_ips[] = {
+      "2001:db8::1",
+      "fe80::abcd:ef65:4321",
+      "fd11:feed:beef:0:cafe::4",
```

Comment:
Could you please add test cases where san_ip has port and/or zone_id?

e.g. "128.10.0.1:8888", "[2001:db8::1]:8080", "[2001:db8::1%em1]:8080"

Reply:
sure!